src/main/java/org/hl7/davinci/priorauth/App.java

```diff
@@ -40,8 +40,16 @@ public static void main(String[] args) {
     if (args.length > 0) {
       if (args[0].equalsIgnoreCase("debug")) {
         debugMode = true;
+        System.out.println("debug mode: arg");
       }
     }
+
+    if (System.getenv("debug").equalsIgnoreCase("true")) {
+      debugMode = true;
+      System.out.println("debug mode: env");
+    }
+
```

Comment:
So this now supports debug mode through command line args or an environment variable called "debug". On HSPC portainer I assume the new environment variable is set?

Reply:
Yes, that is correct, either will set it. I already added the environment variable in Portainer.